mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -662,12 +662,15 @@ def _get_attributes_filtering_clauses(parsed):
         value = sql_statement.get('value')
         comparator = sql_statement.get('comparator')
         if SearchUtils.is_attribute(key_type, comparator):
-            # validity of the comparator is checked in SearchUtils.parse_search_filter()
-            op = SearchUtils.filter_ops.get(comparator)
-            if op:
-                # key_name is guaranteed to be a valid searchable attribute of entities.RunInfo
-                # by the call to parse_search_filter
-                attribute_name = SqlRun.get_attribute_name(key_name)
+            # key_name is guaranteed to be a valid searchable attribute of entities.RunInfo
+            # by the call to parse_search_filter
+            attribute_name = SqlRun.get_attribute_name(key_name)
+            if comparator.lower() in ['like', 'ilike']:
+                op = SearchUtils.get_sql_filter_ops(getattr(SqlRun, attribute_name),
+                                                    comparator.lower())
```

Comment:
Small but adds up: assigning comparator.lower() in a var and reusing that var will be faster than 2x comparator.lower()